### ISSUE TITLE
Rewrite the Zcash Sprout JoinSplit gadget in ZoKrates

### ISSUE THREAD

**Issue by kunxian-xia** [Zokrates/ZoKrates#230]
As I know, currently Zcash Sprout release's joinsplit gadget is only implemented in C++ using libsnark's gadget APIs.

**Comment by Schaeff** [comment]
Agreed, it would be great to have that. @stefandeml this sounds a bit too high level to be in the stdlib in my opinion but would love to hear your thoughts
